test/setup/main.tf

```diff
@@ -14,6 +14,17 @@
  * limitations under the License.
  */
 
+resource "random_string" "suffix" {
+  length  = 6
+  special = false
+  upper   = false
+}
+
+resource "google_folder" "test_folder" {
```

Comment:
I think I've asking this in the past, sorry if you've answered already: this test folder in creating in the CFT-owned org, right? Is  it cleaned up during test shutdown?

Reply:
Yep! All the resources created in `test/setup` are specific to be created during CFT pipeline and tested against the `examples/` folder.
Both `setup` and `examples` resources are created and cleaned/removed in CFT organization during the CI process.